src/angular-app/bellows/shared/sound-player.component.ts

```diff
@@ -16,7 +16,8 @@ export class SoundController implements angular.IController {
 
   $onInit(): void {
 
-    this.slider = this.$element.find('.seek-slider').get(0) as HTMLInputElement;
+    this.slider = document.getElementById('slider') as HTMLInputElement;
```

Comment:
There should be a way to select the element by it's class name, since we are already scoped to the component.  I'm thinking something like:
`this.$element.querySelector('.seek-slider')`
but I don't know if $element is usable anymore.  There must be a way to access the HTML element of the component, from inside the component :)

This is preferable to adding an id on the input, since an id is supposed to be unique to the entire page, we are creating a situation where multiple `id="slider"` could be present on the page, one for each audio file.  We can avoid this by using the existing class name `seek-slider`

Reply:
> There should be a way to select the element by it's class name, since we are already scoped to the component. I'm thinking something like:
this.$element.querySelector('.seek-slider')
but I don't know if $element is usable anymore. There must be a way to access the HTML element of the component, from inside the component :)

yes, depending on whether `$element` is a standard element or some other jqlite version of an element, see
[querySelector()](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector) for the standard api

Reply:
Fixed this morning